packages/caver-klay/caver-klay-accounts/src/account/account.js

```diff
@@ -0,0 +1,70 @@
+const AccountKeyPublic = require('../accountKey/accountKeyPublic')
+const AccountKeyMultiSig = require('../accountKey/accountKeyMultiSig')
+const AccountKeyRoleBased = require('../accountKey/accountKeyRoleBased')
+const isAddress = require('../../../../caver-utils/src/utils').isAddress
+
+class Account {
+	static fromObject(obj) { return new Account(obj.address, new AccountKeyPublic(obj.privateKey)) }
+	
+	static isAccountKey(accountKey) {
+		let isAccountKey = false
+		if (accountKey instanceof AccountKeyPublic) isAccountKey = true
+		if (accountKey instanceof AccountKeyMultiSig) isAccountKey = true
+		if (accountKey instanceof AccountKeyRoleBased) isAccountKey = true
+		
+		return isAccountKey
+	}
+
+	constructor(address, accountKey) {
+		if (!address || !accountKey) throw new Error('Failed to create Account. address and accountKey are needed to create Account.')
+
+		if (!isAddress(address)) throw new Error(`Invalid address : ${address}`)
+		if (!Account.isAccountKey(accountKey)) throw new Error(`Invalid accountKey.`)
+
+		Object.defineProperty(this, 'address', {
+			get: function () { return address },
+			set: function (addressInput) {
+				if (!isAddress(addressInput)) throw new Error(`Invalid address : ${addressInput}`)
+				address = addressInput
+			},
+			enumerable: true
+		})
+
+		Object.defineProperty(this, 'accountKey', {
+			get: function () { return accountKey },
+			set: function (accountKeyInput) {
+				if (!Account.isAccountKey(accountKeyInput) && accountKeyInput !== null) throw new Error(`Invalid accountKey.`)
+				
+				if (accountKeyInput === null) {
```

Comment:
Is this right? `accountKeyInput === null`? instead of `accountKeyInput !== null`?

Reply:
At caver.klay.accounts.wallet.remove function, before delete instance, set null to privateKey(originally).
So after appliance accountKey to remove function, i set null to accountKey.
This logic for that !

Reply:
Is it okay to call accountKey.update() after accountKey = null?

Something like below:
a.accountKey = null
a.AccountKey = newAcc

Reply:
I think for that usage, i should add something here. I added logic for that :)